README.md

```diff
@@ -1181,4 +1181,4 @@ See [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Copyright and License
 
-Copyright (c) 2012-2014 Tim Vandecasteele and contributors. See [LICENSE.txt](LICENSE.txt) for details.
+Copyright (c) 2012-2016 Tim Vandecasteele and contributors. See [LICENSE.txt](LICENSE.txt) for details.
```

Comment:
I got that, but I don't think it's OK still because it would strip Tim from Copyright for 2015-2016 and I don't think that's right. Let me see if I can have an actual attorney comment.